fulp_modules/features/antagonists/bloodsuckers/code/powers/targeted/lunge.dm

```diff
@@ -3,20 +3,27 @@
 	desc = "Spring at your target to grapple them without warning, or tear the dead's heart out. Attacks from concealment or the rear may even knock them down if strong enough."
 	button_icon_state = "power_lunge"
 	power_explanation = "Predatory Lunge:\n\
-		Click any player to, after a short delay, dash at them.\n\
+		Click any player to start spinning wildly and, after a short delay, dash at them.\n\
 		When lunging at someone, you will grab them, immediately starting off at aggressive.\n\
-		There is an exception to this, those wearing Riot gear, and Monster Hunters, will be passively grabbed instead.\n\
+		Riot gear and Monster Hunters are protected and will only be passively grabbed.\n\
 		You cannot use the Power if you are already grabbing someone, or are being grabbed.\n\
+		If you grab from behind, or from darkness (Cloak of Darkness works), you will knock the target down.\n\
 		If used on a dead body, will tear their heart out.\n\
 		Higher levels increase the knockdown dealt to enemies.\n\
-		At level 4, if you grab from behind or from darkness (Cloak of Darkness works), you will knock the target down."
-	power_flags = BP_AM_TOGGLE
+		At level 4, you will no longer spin, but you will be limited to tackling from only 6 tiles away."
+	power_flags = NONE
```

Comment:
how does changing this flag alter its function?

Reply:
The flag causes it to start processing as you click on the button, which starts the spinning. Instead we start processing on the Lunge itself when they click on a target, so they spin when they are actually charging up at someone.

Reply:
feels weird defining things as NONE like this but hey, if it works...